packages/shared/src/regex.ts

```diff
@@ -1,6 +1,6 @@
 export const emailRegEx = /^\S+@\S+\.\S+$/;
 export const phoneRegEx = /^\d+$/;
-export const usernameRegEx = /^[A-Z_a-z-][\w-]*$/;
+export const usernameRegEx = /^[A-Z_a-z]\w*$/;
```

Comment:
Not sure whether a username can start with an underscore?

Reply:
allow for now per Slack thread [discussion](https://silverhand-io.slack.com/archives/C0243P3E4KS/p1656568535592549)